### PR TITLE
add line 1

### PR DESCRIPTION
**Pull request by bicotrucarduino**
add line 1 comment

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+add line 1
```
Comment (VoLinhTruc) [inline README.txt]: are you sure this is correct?

Comment (VoLinhTruc) [reply README.txt]: pls make it sure

Comment (bicotrucarduino) [reply README.txt]: yes 100% sure, man

Comment (VoLinhTruc) [reply README.txt]: and a . at the end of line

Comment (VoLinhTruc) [reply README.txt]: add a - at the beginning of line

**Review by VoLinhTruc — CHANGES_REQUESTED** [review]

File: README.txt [hunk 2]
```diff
@@ -1 +1 @@
-add line 1
+-add line 1.
```
Comment (bicotrucarduino) [inline README.txt]: OK done

Comment (VoLinhTruc) [inline README.txt]: OK good

Comment (VoLinhTruc) [inline README.txt]: review complete

**Review by VoLinhTruc — APPROVED** [review]
review complete

File: README.txt [hunk 3]
```diff
@@ -0,0 +1 @@
...
```
Comment (VoLinhTruc) [inline README.txt]: add line 2 and line 3 with the sme manner

**Review by VoLinhTruc — CHANGES_REQUESTED** [review]

File: README.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+-add line 1.
+-add line 2.
+-add line 3.
```
Comment (bicotrucarduino) [inline README.txt]: add done

Comment (VoLinhTruc) [reply README.txt]: ok let me check

Comment (VoLinhTruc) [reply README.txt]: > add done

ok let me check

Comment (bicotrucarduino) [reply README.txt]: pls hurry

Comment (VoLinhTruc) [reply README.txt]: ok good to merger

**Review by VoLinhTruc — APPROVED** [review]

**Comment by VoLinhTruc** [comment]
done merger

**Pull request merged**